web_source/travellers/ben_smith.qmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "Ben Smith"
+subtitle: "Armchair Adventurer"
+image: https://commons.wikimedia.org/wiki/File:Brad_Pitt_in_2014_%28cropped%29.jpg
+toc: false
+about:
+  id: person-profile
+  template: jolla
+---
+
+```{=html}
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="../travellers.html">Travellers</a></li>
+    <li class="breadcrumb-item active" aria-current="page">{{< meta title >}}</li>
+  </ol>
+</nav>
+```
+
+:::{#person-profile}
+:::
+
+## Biography
+
+FIXME: add a one line description about the person
```

Comment:
Did you miss the one liner description here? I did too but my buddy pointed it out :-)

Reply:
added!  good spot!